src/components/views/elements/ReplyThread.tsx

```diff
@@ -35,6 +37,8 @@ import ReplyTile from "../rooms/ReplyTile";
 import Pill from './Pill';
 import { Room } from 'matrix-js-sdk/src/models/room';
 
+const SHOW_EXPAND_QUOTE_PIXELS = 60;
```

Comment:
A comment to explain how the number is derived would be appreciated, so we know what to set it to when we break something.